code_schemes/facebook_post_type.json

```diff
@@ -0,0 +1,47 @@
+{
+  "SchemeID": "Scheme-3cd48f7c",
+  "Name": "Post Type",
+  "Version": "0.0.0.1",
+  "Codes": [
+    {
+      "CodeID": "code-d5b87090",
+      "CodeType": "Normal",
+      "DisplayText": "picture",
```

Comment:
I thought about photo too, to mirror the Facebook terminology. The only reason I changed to picture was because people could upload images that aren't photographs. So I still have a slight leaning towards "image" or "picture", but I don't mind changing to "photo" if you think we should align with the Facebook API's terminology in our analysis.

Reply:
Okay, I think we should maintain FB terminologies where possible.

Reply:
Updated